pandas/indexes/multi.py

```diff
@@ -2102,7 +2204,9 @@ def equals(self, other):
             return True
 
         if not isinstance(other, Index):
-            return False
+            if not isinstance(other, tuple):
+                return False
+            other = Index([other])
```

Comment:
Is there a reason for this change? 
I can remember we had some discussion about this before whether or not `Index.equals` should coerce it's argument to an index. But it seems a bit strange to do this solely for a single tuple.